src/error_logger_lager_h.erl

```diff
@@ -74,7 +74,7 @@ set_high_water(N) ->
     gen_event:call(error_logger, ?MODULE, {set_high_water, N}, infinity).
 
 -spec init(any()) -> {ok, #state{}}.
-init([HighWaterMark]) ->
+init([{sink, Sink}, HighWaterMark]) ->
```

Comment:
This is no longer relevant: I've special-cased the throttle and no longer break all existing backends.